include/physicsobjects.hxx

```diff
@@ -302,6 +302,28 @@ CutQuantity(ROOT::RDF::RNode df, const std::string &outputname,
         {quantity});
 }
 
+/**
+ * @brief This function returns the length of the vector containers in columns
+ * that contain `ROOT::VecOps::RVec<T>` objects. The function is templated
+ * with `T`, which must be set to the underlying type of the vector object.
+ *
+ * @param df input dataframe
+ * @param outputname name of the output column storing the object count
+ * @param vector_quantity name of the quantity stored in vector structures
+ *
+ * @return a dataframe with a new column
+ */
+template <typename T>
+ROOT::RDF::RNode Size(ROOT::RDF::RNode df, const std::string &outputname,
+                       const std::string &vector_quantity) {
```

Comment:
If you define it in a header and the function is small, it is better to define it with `inline`.
```cpp
inline ROOT::RDF::RNode ...
```